tcmur_cmd_handler.c

```diff
@@ -2220,7 +2312,18 @@ int tcmur_cmd_passthrough_handler(struct tcmu_device *dev,
 	 * handle_passthrough_cbk().
 	 */
 	track_aio_request_start(rdev);
+
+        rcmd_state = calloc(1, sizeof(*rcmd_state));
+	if (!rcmd_state) {
+		tcmu_dev_err(dev, "Failed to calloc memory for tcmur_cmd_state!\n");
+		ret = TCMU_STS_NO_RESOURCE;
+		goto untrack;
+	}
+
+	tcmu_cmd_set_private(cmd, rcmd_state);
 	ret = handle_passthrough(dev, cmd);
+
+untrack:
 	if (ret != TCMU_STS_ASYNC_HANDLED)
 		track_aio_request_finish(rdev, NULL);
```

Comment:
Need to add a

free(rcmd_state);

in this if chunk when handle_passthrough returned a failure.